src/librbd/Utils.cc

```diff
@@ -89,7 +91,7 @@ uint64_t parse_rbd_default_features(CephContext* cct)
         value += conf_vals[feature];
       } else {
         ret = -EINVAL;
-        ldout(cct, 1) << "Warning: unknown rbd feature " << feature << dendl;
+        lderr(cct) << "unknown rbd feature " << feature << dendl;
```

Comment:
Yes, I am aware that execution will continue, but I think it is better to display it as an error message to the user (though not fatal). Without this and with `rbd debug = 0` by default, the user will not see e.g. that she has a typo in ceph.conf.